releasenotes/notes/fix_inequlity_energy-0adf721256a75a30.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fix energy calculation when the lower bound (lb) equals the upper bound (lb)
+    in ``add_linear_equality_constraint``.
```

Comment:
Can we update this to be ``` ``DQM.add_linear_equality_constraint``` ``

Reply:
Or better ``` ``DiscreteQuadraticModel.add_linear_equality_cosntraint`` ```

Reply:
it is for both BQM and DQM. I'll add both `DiscreteQuadraticModel.add_linear_equality_cosntraint`` and `BinaryQuadraticModel.add_linear_equality_cosntraint``

Reply:
You're totally, right. Perfect, thanks.